server/src/main/java/io/druid/server/initialization/jetty/JettyServerInitUtils.java

```diff
@@ -43,6 +43,7 @@ public static GzipHandler wrapWithDefaultGzipHandler(final Handler handler)
     GzipHandler gzipHandler = new GzipHandler();
     gzipHandler.setMinGzipSize(0);
     gzipHandler.setIncludedMethods(GZIP_METHODS);
+    gzipHandler.setInflateBufferSize(4096);
```

Comment:
does the buffer size needs to be configurable ?

Reply:
+1